app/javascript/components/mentoring/mentor-discussion/IterationButton.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { Iteration } from '../MentorDiscussion'
+
+const formatCommentCount = (count: number) => {
+  if (count >= 0 && count <= 9) {
+    return count
+  } else {
+    return '9+'
+  }
```

Comment:
Is the `count >= 0` needed? Currently, if the count is somehow less than zero, it would display `9+`. Maybe invert the logic to make it clear what the exceptional case is?

```suggestion
  if (count > 9) {
    return '9+'
  } else {
    return count
  }
```

Or even more concise:

```suggestion
  return count > 9 ? '9+' : count
```